src/Entity/ServerInterface.php

```diff
@@ -71,4 +71,58 @@ public function getPersistedQueryInstances();
    */
   public function getSortedPersistedQueryInstances();
 
+  /**
+   * Gets disable introspection config.
+   *
+   * @return bool
+   *   The disable introspection config, FALSE otherwise.
+   */
+  public function getDisableIntrospection();
```

Comment:
we should not change the interface at this point because graphql 4.0 has already been released.

Can we only do the changes on the Server class?